src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -2830,6 +2833,7 @@ class C
             End Using
         End Function
 
+        ' <WpfFact, Trait(Traits.Feature, Traits.Features.Completion)>
```

Comment:
@rchande I tried to enable this test but it fails.  Can you take a look and fix/enable the test?